front_end/cvMatch[] tracks 
sensor_msgs/Image motionInliers